uninstall.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ *
+ * This file runs when the plugin in uninstalled (deleted).
+ * This will not run when the plugin is deactivated.
+ * Ideally you will add all your clean-up scripts here
+ * that will clean-up unused meta, options, etc. in the database.
+ *
+ */
```

Comment:
:no_entry_sign: **Error**: Missing @package tag in file comment (*Squiz.Commenting.FileComment.MissingPackageTag*).